cli/package.json

```diff
@@ -116,7 +116,7 @@
     "cypress": "bin/cypress"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": "^14.0.0 || ^16.0.0 || >=18.0.0"
```

Comment:
why are we dropping support for non-LTS versions of Node?

Reply:
They are at EOL. See: [endoflife.date/nodejs](https://endoflife.date/nodejs) & https://cypressio.slack.com/archives/C042BV2KGRJ/p1669752580650829?thread_ts=1669750249.124649&cid=C042BV2KGRJ